homeassistant/components/binary_sensor/rfxtrx.py

```diff
@@ -100,6 +100,11 @@ def binary_sensor_update(event):
             pkt_id = "".join("{0:02x}".format(x) for x in event.data)
             sensor = RfxtrxBinarySensor(event, pkt_id)
             sensor.hass = hass
+            hass.bus.fire(
```

Comment:
I don't think that this makes much sense? One can already just look at state changed events and look at states where "from" is `None`

Reply:
Interesting. Yeah, I guess that should work. This feels a bit more elegant perhaps, but I'll give that a go. I think it should do the trick! Thank you.